examples/affinity/README.md

```diff
@@ -0,0 +1,102 @@
+
+The custom resource also has `spec.affinity.nodeAffinityLabels` to ensure the application pods will run on nodes with the `disktype: ssd` label.
+
+```yaml
+apiVersion: app.stacks/v1beta1
+kind: RuntimeComponent
+metadata:
+  name: coffeeshop-backend
+spec:
+  applicationImage: 'k8s.gcr.io/pause:2.0'
+  replicas: 2
+  affinity:
+    nodeAffinityLabels:
+      disktype: ssd
+    podAntiAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        - labelSelector:
+            matchExpressions:
+              - key: app.kubernetes.io/instance
+                operator: In
+                values:
+                  - coffeeshop-backend
+          topologyKey: kubernetes.io/hostname
+```
+
+The YAML snippet below shows the `RuntimeComponent` CR for the `coffeeshop-frontend` application. The snippet has the `.spec.affinity.podAffinity` configured which informs the scheduler that all its replicas are to be co-located with pods that have selector label `app.kubernetes.io/instance: coffeeshop-backend`. The CR also includes `.spec.affinity.podAntiAffinity` to ensure that each `coffeeshop-frontend` replica does not co-locate on a single node. In addition, the `spec.affinity.nodeAffinityLabels` parameter is specified to ensure the application pods will run on nodes with the `disktype: ssd` label.
+
+```yaml
+apiVersion: app.stacks/v1beta1
+kind: RuntimeComponent
+metadata:
+  name: coffeeshop-frontend
+spec:
+  applicationImage: 'k8s.gcr.io/pause:2.0'
+  replicas: 2
+  affinity:
+    nodeAffinityLabels:
+      disktype: ssd
+    podAntiAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        - labelSelector:
+            matchExpressions:
+              - key: app.kubernetes.io/instance
+                operator: In
+                values:
+                  - coffeeshop-frontend
+          topologyKey: kubernetes.io/hostname
+    podAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        - labelSelector:
+            matchExpressions:
+              - key: app.kubernetes.io/instance
+                operator: In
+                values:
+                  - coffeeshop-backend
+          topologyKey: kubernetes.io/hostname
+```
+
+To see where the which nodes the pods are running, run the following command:
+
+```console
+oc get pods -o wide
+```
+
+And the output would look like the following:
+
+```console
+NAME                                   READY   STATUS    RESTARTS   AGE     IP              NODE
+coffeeshop-backend-f57557d99-5lm24     1/1     Running   0          6m8s    10.254.20.173   worker0.shames.os.fyre.ibm.com
+coffeeshop-backend-f57557d99-fv62n     1/1     Running   0          6m8s    10.254.5.39     worker1.shames.os.fyre.ibm.com
+coffeeshop-frontend-789c585488-5k7sh   1/1     Running   0          5m52s   10.254.5.38     worker1.shames.os.fyre.ibm.com
+coffeeshop-frontend-789c585488-9m529   1/1     Running   0          5m52s   10.254.20.172   worker0.shames.os.fyre.ibm.com
+```
+
```

Comment:
Summarizing the results of this particular example here would be good. Something like: 
"As you can see, there is only one instance of coffeeshop-backend or coffeeshop-frontend running on one node, as defined by pod anti-affinity. Also, an instance of coffeeshop-backend and an instance of coffeeshop-frontend are running together on the same node, as defined by pod affinity."

Reply:
Done in https://github.com/application-stacks/runtime-component-operator/pull/133/commits/06f4c1014e6f595740b3a94c26770547e0fd7cf7